examples/hardhat/hardhat.config.ts

```diff
@@ -1,14 +1,15 @@
 require('dotenv').config()
 import '@nomiclabs/hardhat-ethers'
-import { HardhatUserConfig } from 'hardhat/types'
+
+import { HardhatUserConfig } from 'hardhat/config'
 
 const config: HardhatUserConfig = {
   solidity: '0.8.4',
   networks: {
     hardhat: {
       forking: {
         enabled: true,
-        url: process.env.RPC_URL!,
+        url: process.env.ARCHIVE_NODE_RPC_URL!,
```

Comment:
@krzkaczor could you set this in GitHub Acations secrets? (We can use my url, but I don't have perms to edit settings)

Reply:
I don't want to use secrets here b/c then tests will fail on forks. And if we forward secrets automatically to external repo we can expose something by accident. I think the only way forward is to add it explicitly in the CI config.

Reply:
All right, I'm adding a default here. If it exceeds the limit the developer can pass an env var.

Reply:
> I don't want to use secrets here b/c then tests will fail on forks.

To be fair, I actually skipped the test on fork branches and PRs from forks run with PR target secrets, don't they?

I just removed the following, but I'm a tad conflicted about it.
![image](https://user-images.githubusercontent.com/15332326/142927885-c34502a4-0c8e-478d-99f4-51ea2473f2b3.png)

We could just skip the test if the secret isn't passed, especially since it's a "end-to-end" kind of thing.